vnext/ReactUWP/EndPoints/dll/react-native-uwp.x64.def

```diff
@@ -81,3 +81,15 @@ nallocxWeak
 mallctlWeak
 mallctlnametomibWeak
 mallctlbymibWeak
+
+?Value@ValueAnimatedNode@uwp@react@@QEAANXZ
```

Comment:
Why do all of these need to be exported? -- Is this just for UTs?  Any way we can have the UTs link in the code directly, so we can keep the exported surface small.  -- At some point we have to go over the entire exported surface and ensure it is all ABI/winrt.  So I'd like to keep us honest about what the surface is.